fsButler/utils.py

```diff
@@ -315,14 +333,18 @@ def strictMatch(cat1, cat2, matchRadius=1*afwGeom.arcseconds, includeMismatches=
                 bestMatches[id] = (m1, m2, d)
 
     if includeMismatches:
-        print "{0} objects from {1} in the first catalog had no match in the second catalog.".format(len(noMatch), len(cat1))
-        print "{0} objects from the first catalog with a match in the second catalog were not the closest match.".format(len(matched) - len(noMatch) - len(bestMatches))
+        print "{0} objects from {1} in the first catalog had no match in the second\
+        catalog.".format(len(noMatch), len(cat1))
```

Comment:
(Here and below) in Python 3 (or Python 2 with `from __future__ import print_function`, which is the LSST standard), this is a syntax error.